### ISSUE TITLE
Whether int8 quantization is supported ？

### ISSUE THREAD

**Issue by tensorflowt** [TrojanXu/yolov5-tensorrt#42]
May I ask whether the current project supports INT8 quantization? If so, how? Currently onlyFT16, FT32 quantification is supported, right?

**Comment by TrojanXu** [comment]
Correct, but I think you can do PTQ using TensorRT calibration. Not implemented in this repo and not planned to do so in short phase.
